Code/Legacy/CryCommon/LoadScreenComponent.cpp

```diff
@@ -42,10 +42,6 @@ namespace
 
 #if defined(CARBONATED)
     static const char* const s_levelLoadScreenUiCanvasPath = "level_load_screen_uicanvas_path";
-
-    static const char* const s_globalLevelFixedFpsCvarName = "global_level_load_screen_sequence_fixed_fps";
-    static const char* const s_globalLevelMaxFpsCvarName = "global_level_load_screen_max_fps";
-    static const char* const s_globalLevelMinimumLoadTimeCvarName = "global_level_load_screen_minimum_time";
```

Comment:
Why was this removed?  We -should- be defining the global load screen (as a default)

Reply:
They are described in code but never used in LY game.
Ok, I have return them back.